charts/dex.yaml

```diff
@@ -0,0 +1,56 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/infra: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: '/$2'
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    kubernetes.io/ingress.class: "nginx-control-plane"
+  path: /oidc(/|$)(.*)
+  hosts:
+    - null
```

Comment:
Control-plane IP here?

Reply:
Not sure we can have an IP address in ingress's host

Reply:
No, not control plane IP... I'm not even sure `null` is right, maybe the lists needs to be empty?

In any case, we want to be the 'fallback' entry (like we do for UI, Grafana,...) for the ingress controller.

Reply:
Unfortunately, this may be hard to do using the Helm chart, given this [template](https://github.com/helm/charts/blob/master/stable/dex/templates/ingress.yaml#L27-L35):

```yaml
  rules:
  {{- range .Values.ingress.hosts }}
    - host: {{ . | quote }}
      http:
        paths:
          - path: {{ $ingressPath }}
            backend:
              serviceName: {{ $fullName }}
              servicePort: {{ $servicePort }}
  {{- end }}
```